### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by gaks1**
1. Created simple HTML to add and remove books.
2. Create javascript to add, remove and store books locally.
3. Created simple css for the hr element.
4. Added linters.

### CODE REVIEW

**Review by laiifuu — COMMENTED** [review]
# Status: Invalidated :stop_sign: 

## Hi team :wave:  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You have submitted a project that doesn't have a descriptive README. You used the template provided by Microverse, however, you need to fill it with your own information.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by gaks1** [comment]
@laiifuu  Hello but we have descriptive readme  that has our information on this branch

**Comment by laiifuu** [comment]
@gaks1, I appologize I made a mistake. I'm terribly sorry. In that case you can go ahead and re submit for another review. Since it was an invalidation, your number of reviews should be intact. Again, I'm very sorry about this.

File: README.md [hunk 1]
```diff
@@ -60,18 +58,14 @@ After you're finished please remove all the comments and instructions!
...
+**[Awesome Books]** is a simple that adds or removes books.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
   <summary>Client</summary>
```
Comment (nahnahdev) [inline README.md:69]: - [ ] Kindly replace this section with the right information. This is how it currently looks
- 
image

File: script.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  bookData.author = author.value;
+
+  titleInput.value = '';
+  author.value = '';
+  bookArray.push(bookData);
+  stringifydata = JSON.stringify(bookArray);
+  localStorage.setItem('storeddata', stringifydata);
+  bookContainer.innerHTML = '';
+  setValue();
+}
```
Comment (nahnahdev) [inline script.js]: - [ ] It looks like there is a problem with your function to add a book. As a result of this, I am unable to add books
 to your app . I have attached a video below for your reference :100: 
[Screencast from 02-13-2023 05:52:22 PM.webm](https://user-images.githubusercontent.com/102699958/218536308-98fdc187-fdcb-42d0-ad8a-264924284615.webm)
- [ ] It throws this error in the console so kindly fix it. As a result of this I won't be able to test the other  functionalities so the next reviewer will check them before your project is approved :+1: 
image

**Review by nahnahdev — CHANGES_REQUESTED** [review]
Hi @gaks1 and @amanuelbelete95 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- [ ] No linter errors :heavy_check_mark: 
- [ ] Good use of GitHub flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  titleInput.value = '';
+  author.value = '';
+  bookArray.push(bookData);
+  const bookArrayString = JSON.stringify(bookArray);
+  localStorage.setItem('storeddata', bookArrayString);
+  const t = localStorage.getItem('storeddata');
+  const parseddata = JSON.parse(t);
+  bookContainer.innerHTML = '';
+  setValue(parseddata);
+}
```
Comment (ABDULALI3468) [inline script.js]: As requested by the Previous Code Reviewer the problem still exists with your function and user are not able to add a book.  

After seeing it a bit I found that the problem is with the `bookArrayString1` it is not presented when the page loads. Consider adding it as an empty array to the local storage when the page loads and then add the further objects in it. There may be some other reasons and problems with the function so consider rechecking it as they still exist.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @amanuelbelete95 , 



Great work on making the changes requested by a previous reviewer 👏 
You've done really well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @amanuelbelete95 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**